### PR TITLE
Update docs to clarify which Supervisor port to expose outside cluster

### PR DESCRIPTION
**Pull request by cfryanr**
Update docs to clarify which Supervisor port to expose outside cluster.

This PR also increases the font size of pre formatted code blocks, which was recently decreased to a small font size (by mistake?) in another PR.

This PR also changes the name of this document in the left-hand sidebar. It was recently changed to "As an OIDC Provider" in another PR. This PR changes it to "Required Configuration" in an attempt to make it more clear that the configuration described in this doc are typically the first things that you would configure after installing the Supervisor and that it is not optional for a Supervisor installation.

See deploy preview here: https://deploy-preview-1631--pinniped.netlify.app/docs/howto/supervisor/configure-supervisor/ (until this PR is merged).

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by joshuatcasey — APPROVED** [review]

**Review by benjaminapetersen — APPROVED** [review]
👍

**Review by joshuatcasey — APPROVED** [review]

**Comment by cfryanr** [comment]
Thanks to @djpbessems for providing suggestions and feedback for this PR!

**Comment by benjaminapetersen** [comment]
Noting quick that we have a few font size issues on the live site that we can fix.  Quick screenshots:
Screenshot 2023-09-08 at 12 56 56 PM
vs
Screenshot 2023-09-08 at 12 56 59 PM
Going to do a follow-on and fix.

File: site/themes/pinniped/assets/scss/_components.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@ code {
 }
 pre code {
     background: none;
+    font-size: 100%;
```
Comment (benjaminapetersen) [inline site/themes/pinniped/assets/scss/_components.scss:12]: I'll swing back around and take a look at this.

**Pull request merged**